app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+
+def copy_file(command: str) -> None:
```

Comment:
The function should verify that the input string `command` contains exactly three elements and starts with 'cp'. This is a requirement according to the task description and checklist.